app/views/accounts/show.rabl

```diff
@@ -0,0 +1,8 @@
+extends 'activitypub/types/person.rabl'
```

Comment:
It could be helpful to name this files `show.activitystreams2.rabl` because e.g. I would be confused what they're used for otherwise (name of file dot format dot processing language)